pages/en/lb3/Polymorphic-relations.md

```diff
@@ -96,47 +138,140 @@ And:
 
 {% include code-caption.html content="common/models/reader.js" %}
 ```javascript
-Reader.hasMany(Picture, { polymorphic: { // alternative syntax  
-  as: 'imageable', // if not set, default to: reference
-  foreignKey: 'imageableId', // defaults to 'as - Id'
-  discriminator: 'imageableType' // defaults to 'as - Type'
+Reader.hasMany(Picture, { polymorphic: {
+  foreignKey: 'imageableId',
+  discriminator: 'imageableType'
+  } 
+});
+
+// Alternative use `selector`
+
+Reader.hasMany(Picture, { polymorphic: {
+  selector: "imageable"
   } 
 });
 ```
 
-## HasManyThrough polymorphic relations
+## BelongsTo polymorphic relations
 
-To define a hasMany polymorphic relation, there must be a "through" model, similarly to a standard [HasManyThrough relation](HasManyThrough-relations.html).
+Because you define the related model dynamically, you cannot declare it up front.
+So instead of passing in the related model (name), you specify the name of the polymorphic relation.
```

Comment:
Missing comma after "so":

`So, instead of passing in the related model (name)...`